src/main/java/com/patsi/bean/SocialMediaUser.java

```diff
@@ -7,31 +7,35 @@
 import jakarta.persistence.EnumType;
 import jakarta.persistence.Enumerated;
 import jakarta.persistence.Id;
-import lombok.Builder;
-import lombok.Data;
+import lombok.*;
 
 import java.util.List;
 import java.util.UUID;
 
 @Entity
 @Data
 @Builder
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class SocialMediaUser {
     @Id
     private UUID uid;
     //For people to search
     private String userName;
     private int userNameChangeCount;
     private String displayName;
+    private String email;
     private String profilePicture;
-    private String BannerPicture;
+    private String bannerPicture;
+    @Builder.Default
     @Enumerated(EnumType.ORDINAL)
     private AccountStatus accountStatus = AccountStatus.Inactive;
+    @Builder.Default
     @Enumerated(EnumType.ORDINAL)
     private AccountType accountType = AccountType.privateAccount;
     private String biography;
-    private int followersCount;
-    private int followingCount;
+    private int followersCount = 0;
```

Comment:
U may need @Builder.Default for default value

Reply:
fixed